content/en/tracing/trace_collection/custom_instrumentation/go/otel.md

```diff
@@ -164,6 +147,29 @@ ctx, span := t.Start(
 span.End()
 ```
 
+## Adding Span Events
```

Comment:
Hey Brett, so since this is a feature I thought it made sense to have it upper case(lower case for me looks terrible) and also matches OTEL docs: https://opentelemetry.io/docs/languages/ruby/instrumentation/#add-span-events

But I guess we have a policy for titles to be all lower case aside from the first word or something like that?

Reply:
Hey, good question. Datadog style is to use sentence case for our headings.

If "span event" is a proper noun, then we could capitalize it in the heading, but then we should also capitalize it in the content too. It looks like OTel docs are inconsistent in how they capitalize the term (sometimes it's "Span Events" and sometimes it's "span events").

Either way works for me, but we should be consistent.

Reply:
Then I'll just leave it like the rest all lower cased, thank you.

Reply:
Hey @brett0000FF please let me know what you think based on the latest commit: https://github.com/DataDog/documentation/pull/25100/commits/37c3e765db65bed00beac9170d38b6a94e4e0d77 and if mentioning the parameters twice is misleading and we should change it somehow, thank you.